### ISSUE TITLE
Selected Color not updating the color in the panel

### ISSUE THREAD

**Issue by jensdc92** [tiaguinho/material-community-components#14]
When I update the selected color it would be nice the color in the panel is updated too. So when i'm updating multiple entities the color of the previous one doesnt stay behind. 

Also when I'm trying to select a color, often I can't select a color by just clicking but i have to drag. 

Thansk for your great work !

**Comment by tiaguinho** [comment]
Hi @jensdc92, when you say: 

> When I update the selected color it would be nice the color in the panel is updated too. So when i'm updating multiple entities the color of the previous one doesnt stay behind.

You mean update the color inside the select area?

**Comment by jensdc92** [comment]
@tiaguinho Yes, that's what I mean

**Comment by dysbulic** [comment]
This is tripping me up as well. I'm using Firestore for my data storage and there's a brief lag between instantiating the component and the color coming in.

Is there a way to get a reference to the `MccColorPicker` using the `@ViewChild` directive? I figured I'd update the color manually, but the class isn't included in the module exports.

**Comment by motabass** [comment]
I really think this is a duplicate of https://github.com/tiaguinho/material-community-components/issues/57

**Comment by motabass** [comment]
I am still not sure what the expeceted behaviour is here exactly

**Comment by motabass** [comment]
...i think i got it now. What @jensdc92 meant was that when while changing the color in the color picker also the color here 
![image](https://user-images.githubusercontent.com/4824274/88743850-0a5cef00-d146-11ea-8e28-d1f75a1eb754.png) should be updated imediatly

**Comment by tiaguinho** [comment]
Can't be this @motabass 
Is change only after confirm or if buttons are disabled.
I'm trying to fix this issue.

**Comment by tiaguinho** [comment]
@motabass I think the refactor you have done fix this issue.
What do you think?

**Comment by motabass** [comment]
I am still not sure what the expected behaviour is?

**Comment by motabass** [comment]
Anyone can confirm if this is fixed in release 9.0.0?
